pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -7,7 +7,7 @@
 , lmdb
 , lmdbxx
 , tweeny
-, mkDerivation
+, wrapQtAppsHook
```

Comment:
I wonder why you do that?

Reply:
There's no need for that,I was confused between the effect of `libsForQt5.callPackage` and the fact `qt5` has it's own `mkDerivation`.